rllib/BUILD

```diff
@@ -1604,21 +1604,21 @@ py_test(
 py_test(
     name = "connectors/tests/test_connector",
     tags = ["team:rllib", "connector"],
-    size = "small",
```

Comment:
do you need to bump these tests sizes. Are your tests timing out now? can you link in bk

Reply:
They have been failing with timeouts, but we could do this in another PR.
Reversed to small for now. Let's see what happens.